src/main/java/mate/academy/springboot/swagger/service/ProductServiceImpl.java

```diff
@@ -0,0 +1,44 @@
+package mate.academy.springboot.swagger.service;
+
+import java.math.BigDecimal;
+import java.util.List;
+import mate.academy.springboot.swagger.model.Product;
+import mate.academy.springboot.swagger.repository.ProductRepository;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.stereotype.Service;
+
+@Service
+public class ProductServiceImpl implements ProductService {
+    private final ProductRepository productRepository;
+
+    public ProductServiceImpl(ProductRepository productRepository) {
```

Comment:
you have already used this @RequiredArgsConstructor annotation in controller. Lets try to adhere unity standart)